release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -397,3 +397,124 @@ Issued: 2021-xx-xx
 Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
 
 link:https://access.redhat.com/solutions/<ARTICLE_ID>[{product-title} 4.7.0 container image list]
+
+[id="ocp-4-7-ibm-z"]
+==== IBM Z and LinuxONE
+
+With this release, IBM Z and LinuxONE is now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster on IBM Z and LinuxONE] for installation instructions.
+
+With this release, KVM on  {op-system-base} 8.3 or later is supported as a hypervisor for user-provisioned installation of {product-title} {product-version} on IBM Z and LinuxONE.  See xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE] for installation instructions.
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Log forwarding
+** Precision Time Protocol (PTP) hardware
+** CSI volume snapshots
+
+* The following {product-title} features are unsupported:
+** {product-title} Virtualization
+** Red Hat OpenShift Service Mesh
+** CodeReady Containers (CRC)
+** {product-title} Metering
+** Multus CNI plug-in
+** FIPS cryptography
+** Encrypting data stored in etcd
+** Automatic repair of damaged machines with machine health checking
+** Tang mode disk encryption during {product-title} deployment
+** {product-title} Serverless
+** Helm command-line interface (CLI) tool
+** Controlling overcommit and managing container density on nodes
+** etcd cluster Operator
+** CSI volume cloning
+** NVMe
+
+* Worker nodes must run {op-system-first}.
+* Persistent shared storage must be provisioned by using either NFS or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or LSO with DASD/FCP.
+* These features are available only for {product-title} on IBM Z for {product-version}:
+** HyperPAV enabled on IBM System Z /LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+===== Supported features
+
+With this release, the following features are supported on IBM Z and LinuxONE:
+
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* OpenShift Do (odo)
+* OpenShift Pipelines TP
```

Comment:
@ktania46 Please add: 
* OVNKube
to the list of supported features. Sorry forgot it in my input.